packages/core/src/components/Copy/index.tsx

```diff
@@ -54,10 +56,10 @@ export default class Copy extends React.Component<Props, State> {
   };
 
   render() {
-    const { prompt, children, id, underlined } = this.props;
+    const { prompt, children, id, trackingName, underlined } = this.props;
     const element = children || (
       // eslint-disable-next-line jsx-a11y/anchor-is-valid
-      <Link id={id}>
+      <Link data-tracking-name={trackingName} id={id}>
```

Comment:
`trackingName={trackingName}`? Both `Link` and `Button` should support it.

Reply:
agreed - will need to at it to ButtonOrLink https://github.com/airbnb/lunar/blob/master/packages/core/src/components/private/ButtonOrLink.tsx#L7

Reply:
@stefhatcher indeed I added to ButtonOrLink to make the prop available across all the ButtonOrLink consumers.